pkg/model/fileshareproto/response.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2019 OpenSDS Authors. All Rights Reserved.
+//
```

Comment:
change "Copyright (c) 2019 OpenSDS Authors. All Rights Reserved. " to "Copyright 2019 The OpenSDS Authors."